test/HydraChain/DaoIncentive.test.ts

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable node/no-extraneous-import */
+import { loadFixture } from "@nomicfoundation/hardhat-network-helpers";
+import { expect } from "chai";
+
+import { EPOCHS_YEAR, ERRORS } from "../constants";
+import { ethers } from "hardhat";
+
+export function RunDaoIncentiveTests(): void {
+  describe("distributeVaultFunds", function () {
```

Comment:
ensure there would be reward bigger than zero when there are only two validators with the minimal possible stake.